source/index.md

```diff
@@ -150,6 +150,7 @@ Or just write Fortran software for your research, business, or schoolwork. You c
 :::{toctree}
 :hidden:
 
+Fortran Playground <https://play.fortran-lang.org/>
```

Comment:
Repeating Fortran here should not be necessary

```suggestion
Playground <https://play.fortran-lang.org/>
```

Reply:
👍